### ISSUE TITLE
Person card custom zones do not work

### ISSUE THREAD

**Issue by gazpachoking** [UI-Lovelace-Minimalist/UI#41]
When using the person card with custom zones defined, the card will crash with the following error if the person is in one of the custom zones:
```
ButtonCardJSTemplateError: TypeError: states[variables.ulm_card_person_zone1] is undefined in 'if (states[variables.ulm_card_person_entity].state != 'home'){ if (states[variables.ulm_card_p...'
```

Taking a peek at the code, it looks like this is because the custom zones defined (ulm_card_person_zone1, and ulm_card_person_zone2) need to be the friendly name of the zones to match the persons state, but then the zone entity is tried to look up with just the friendly name. Suggested fix would be to have the ulm_card_person_zone1 variables be defined as the actual zone, i.e. `zone.work`, then tweak the icon lookup function to something like this: (beware, I'm not a js developer, so it might need some further tweaks)
```javascript
        if (states[variables.ulm_card_person_entity].state != 'home'){
          if (states[variables.ulm_card_person_entity].state == states[variables.ulm_card_person_zone1].attributes.friendly_name){
            var icon = states[variables.ulm_card_person_zone1].attributes.icon != null ? states[variables.ulm_card_person_zone1].attributes.icon : 'mdi:help-circle'
            return '<ha-icon icon="' + icon + '" style="width: 10px; height: 10px; color: var(--primary-background-color);"></ha-icon>';
          } else if (states[variables.ulm_card_person_entity].state == states[variables.ulm_card_person_zone2].attributes.friendly_name){
            var icon = states[variables.ulm_card_person_zone2].attributes.icon != null ? states[variables.ulm_card_person_zone2].attributes.icon : 'mdi:help-circle'
            return '<ha-icon icon="' + icon + '" style="width: 10px; height: 10px; color: var(--primary-background-color);"></ha-icon>';
          } else {
            return '<ha-icon icon="mdi:home-minus" style="width: 10px; height: 10px; color: var(--primary-background-color);"></ha-icon>';
          }
        } else {
          return '<ha-icon icon="mdi:home-variant" style="width: 10px; height: 10px; color: var(--primary-background-color);"></ha-icon>';
        }
```

**Comment by gazpachoking** [comment]
I tested this change and it seems to work.

**Comment by CM000n** [comment]
Thanks for your investigation! That should be fixed with: https://github.com/UI-Lovelace-Minimalist/UI/pull/87

**Comment by CM000n** [comment]
Closed in v1.0.0-beta.8 🏳️‍🌈
